sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -485,4 +489,9 @@ public static CosmosUser createCosmosUser(CosmosAsyncUser asyncUser, CosmosDatab
     public static ConsistencyLevel fromServiceSerializedFormat(String consistencyLevel) {
         return ConsistencyLevel.fromServiceSerializedFormat(consistencyLevel);
     }
+
+    @Warning(value = INTERNAL_USE_ONLY_WARNING)
+    public static CosmosException createServiceUnavailableException(Exception innerException) {
```

Comment:
I think generally, it is better to create more specific exception compared to CosmosExceptions. 
(Another example is when create BadRequestExceptions)

Reply:
Yes, it is. We can incorporate those changes after GA.